setup.py

```diff
@@ -47,7 +47,7 @@
     'cwl-utils==0.5 ; python_version>="3"',
     "pyOpenSSL>=19.0.0",  # FIXME remove once yadage-schemas solves deps.
     "jsonpointer>=2.0",
-    "reana-commons>=0.8.0a13,<0.9.0",
+    "reana-commons[yadage]>=0.8.0a13,<0.9.0",
```

Comment:
Should we remove `yadage-schemas` dep?

I also see many deps that likely depend on `yadage-schemas` (# FIXME remove once yadage-schemas solves deps.), should we move those to reana-commons? Or maybe give it a try if those dep issues are solved.

Reply:
Nicely spotted! Indeed after some testing it seems only `reana-commons[yadage]` is enough